### ISSUE TITLE
[Bug]: Issue installing Coolify 4.0.0-beta.373 on Ubuntu 24.04

### ISSUE THREAD

**Issue by ldesomer** [coollabsio/coolify#4436]
### Error Message and Logs

I encountered an issue while trying to install Coolify 4.0.0-beta.373 on Ubuntu 24.04. The installation stops unexpectedly at step 9 after displaying the joke. The terminal output ends with a hash-like string, with no further progress or clear error message.

```
Algorithm: A word used by programmers when they don't want to explain how their code works.
d713574e9603d45c65668ddea8cbeda2acd9293576e8a6e25dc826e6ac2b9b6e
 ```

### Steps to Reproduce

1. On an Ubuntu 24.04 system, run the following commands:
```bash
curl -fsSL https://cdn.coollabs.io/coolify/install.sh | bash
   ```
 
 2. Observe the output during the installation process. The script halts after step 9, and the terminal displays the following:
 ```
Algorithm: A word used by programmers when they don't want to explain how their code works.
d713574e9603d45c65668ddea8cbeda2acd9293576e8a6e25dc826e6ac2b9b6e
 ```
 
 ### Expected Behavior
 The installation process should proceed without interruption and complete successfully.
 
 ### Actual Behavior
 The installation halts after displaying a random string, with no additional logs or indications of failure.
 
 ### Environment
 - **Operating System:** Ubuntu 24.04  
 - **Docker Version:** 27.0  
 - **Coolify Version:** 4.0.0-beta.373  
 - **Helper Version:** 1.0.4  
 - **Realtime Version:** 1.0.5  
 
### Additional Information
 The issue persists even when using the alternate installation method:  
 ```bash
git clone https://github.com/coollabsio/coolify.git
cd coolify/scripts
./install.sh
  ```

### Coolify Version

4.0.0-beta.373

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Ubuntu 24.04

### Additional Information

- Got the same issue on Debian 12
- VPS hosted by OVH (8 cores, 16GB RAM, 320GB SSD)

**Comment by djsisson** [comment]
@ldesomer can you run
```
docker run -it ubuntu bash
```

**Comment by ldesomer** [comment]
@djsisson thanks for the quick reply, below is the response.
```
Unable to find image 'ubuntu:latest' locally
docker: Error response from daemon: toomanyrequests: Too Many Requests (HAP429).
```

**Comment by ldesomer** [comment]
Indeed, I seem to have exceeded the rate limit!

```
"Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    28    0    28    0     0     98      0 --:--:-- --:--:-- --:--:--    98
jq: parse error: Invalid numeric literal at line 1, column 4
```
I'll try a fresh install after creating an account on docker hub and keep you updated!

**Comment by djsisson** [comment]
@ldesomer after creating you will need to run docker login

**Comment by ldesomer** [comment]
Still have an issue.

```
docker login -u ldesomer
Password: 
Error response from daemon: login attempt to https://registry-1.docker.io/v2/ failed with status: 429 Too Many Requests
```

**Comment by djsisson** [comment]
@ldesomer  try
```sudo systemctl restart docker.socket docker.service```

**Comment by ldesomer** [comment]
It was still not working.

The only workaround I found was to disable IPv6.

```
sudo  nano /etc/sysctl.conf 
```
Adding
```
# Disable IPv6
net.ipv6.conf.all.disable_ipv6 = 1
net.ipv6.conf.default.disable_ipv6 = 1
net.ipv6.conf.lo.disable_ipv6 = 1
```
Then
```
sudo sysctl -p
```

I know the issue isn't on Coolify side but I'd great to add a word about IPv6 issue with Docker in the documentation!

**Comment by peaklabs-dev** [comment]
@ldesomer But your server had an Ipv4 address, right, because at the moment the installer pulls from ghrc.io, which does not support Ipv6 only?

**Comment by ArthurYdalgo** [comment]
so... I had the same issue using the quick installation. I tried the manual one, and eventually I got a log saying port 6001 was already being used (which actually was because the server was being used to host a Soketi instance). We changed it to 6003 and the manual install worked. Might not be the same fix for you dear reader who is facing the same issue, but maybe the manual install will also tell you what is wrong.
